yarn.lock

```diff
@@ -16998,16 +16998,6 @@ react-textarea-autosize@^8.3.0:
     use-composed-ref "^1.0.0"
     use-latest "^1.0.0"
 
-react-transition-group@^2.0.0:
-  version "2.9.0"
-  resolved "https://registry.yarnpkg.com/react-transition-group/-/react-transition-group-2.9.0.tgz#df9cdb025796211151a436c69a8f3b97b5b07c8d"
-  integrity sha512-+HzNTCHpeQyl4MJ/bdE0u6XRMe9+XG/+aL4mCxVN4DnPBQ0/5bfHWPDuOZUzYdMj94daZaZdCCc1Dzt9R/xSSg==
-  dependencies:
-    dom-helpers "^3.4.0"
-    loose-envify "^1.4.0"
-    prop-types "^15.6.2"
-    react-lifecycles-compat "^3.0.4"
-
 react-transition-group@^4.3.0:
```

Comment:
It looks like `react-transition-group` is still here because of something else. Might be a cyclical thing where it's an older version of an otp-ui package we just need to update, but I'd still do an analysis just to double check! I believe I've used `npm why` in this repo before and it worked at least sort of

Reply:
There might be a `yarn why` or similar as well

Reply:
Good catch

Reply:
Actually, React Transition Group is a separate library from React that just helps you do CSS animations by applying and removing CSS classes to your elements. I wouldn't even be surprised if Animate Height package is depending on it. So I think it's fine that it's in here, as long as Velocity is gone.

Reply:
Thanks for double checking!